internal/ui/settings_ui.go

```diff
@@ -378,6 +378,25 @@ func getPicturesDir() (fyne.ListableURI, error) {
 		return nil, err
 	}
 
-	uri := storage.NewFileURI(filepath.Join(home, "Pictures"))
-	return storage.ListerForURI(uri)
+	if runtime.GOOS == "darwin" {
+		uri, err := storage.Child(storage.NewFileURI(home), "Pictures")
+		if err != nil {
+			return nil, err
+		}
+
+		return storage.ListerForURI(uri)
+	}
+
+	const xdguserdir, pic = "xdg-user-dir", "PICTURES"
+	cmd := exec.Command(xdguserdir, pic)
+	stdout := bytes.NewBufferString("")
+	cmd.Stdout = stdout
+	if err = cmd.Run(); err != nil {
+		return nil, err
+	}
+
+	loc := stdout.String()
+	loc = loc[:len(loc)-1] // Remove \n at the end of string
+
+	return storage.ListerForURI(storage.NewFileURI(loc))
```

Comment:
Note lastly that xdg-user-dir will fall back to the home directory, so should check to see if that is the case and fall back to a better default

Reply:
Good point.